scripts/rebuild-ui.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+cd ../jaeger-ui
+
+if git describe --exact-match --tags $(git rev-parse HEAD) >/dev/null 2>&1; then
+    
+    # Get the release version from the tag  
+    release_version = $(git describe --exact-match --tags $(git rev-parse HEAD)) 
+
+
+    # Check if the corresponding UI release has the assets.tar.gz file uploaded
+    release_url="https://github.com/jaegertracing/jaeger-ui/releases/download/${release_version}/assets.tar.gz"
```

Comment:
I wouldn't recommend using the output from git directly like that. Instead, match it against the release tag patter (vN.M.m) and only then use in the URL